nullaway/src/main/java/com/uber/nullaway/dataflow/NullnessStore.java

```diff
@@ -235,15 +235,14 @@ public NullnessStore uprootAccessPaths(
       Map<LocalVariableNode, LocalVariableNode> localVarTranslations) {
     NullnessStore.Builder nullnessBuilder = NullnessStore.empty().toBuilder();
     for (AccessPath ap : contents.keySet()) {
-      if (ap.getRoot().isReceiver()) {
+      Element element = ap.getRoot();
+      if (element == null) {
         continue;
       }
-      Element element = ap.getRoot().getElement();
       for (LocalVariableNode fromVar : localVarTranslations.keySet()) {
         if (element.equals(fromVar.getElement())) {
           LocalVariableNode toVar = localVarTranslations.get(fromVar);
-          AccessPath newAP =
-              new AccessPath(new AccessPath.Root(toVar.getElement()), ap.getElements());
+          AccessPath newAP = AccessPath.switchRoot(ap, toVar.getElement());
```

Comment:
This is a bug fix; before we would not preserve any map-get argument in the new access path